grades/factories.py

```diff
@@ -61,8 +61,8 @@ class Meta:  # pylint: disable=missing-docstring,no-init,too-few-public-methods,
         model = ProctoredExamGrade
 
 
-class MicromastersCertificateFactory(DjangoModelFactory):
-    """Factory for MicromastersCertificate"""
+class MicromastersCourseCertificateFactory(DjangoModelFactory):
```

Comment:
These certificates and edX certificates serve different purposes. There isn't a scenario where we would want to retrieve a `MicromastersCourseCertificate` or an edX `Certificate` instead